larndsim/light_sim.py

```diff
@@ -622,30 +622,66 @@ def sim_triggers(bpg, tpb, signal, signal_op_channel_idx, signal_true_track_id,
     
     return digit_signal, digit_signal_true_track_id, digit_signal_true_photons
 
-def export_light_wvfm_to_hdf5(event_id, waveforms, output_filename, waveforms_true_track_id, waveforms_true_photons, i_mod=-1):
+def zero_suppress_waveform_truth(waveforms_true_track_id, waveforms_true_photons, i_evt, i_trig, i_mod=-1):
+    """
+    Filter empty light waveform backtracking which is filled with the default value '-1', and flatten the backtracking info to 1D array.
+
+    Args:
+        i_evt(int): event id
```

Comment:
The `event_id` refers to a true event. To be inclusive, it could be a spill, a cosmic event or a particle bomb/gun event. The use of `event` seem to be consistent from larnd-sim and onwards.